### PR TITLE
data: mehmetsaidozsoy

### PR DESCRIPTION
**Pull request by mehmetsaidozsoy**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3653"></a>

### CODE REVIEW

File: mehmetsaidozsoy.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/mehmetsaidozsoy_/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/mehmetsaidozsoy",
+      "icon": "FaInstagram"
```
Comment (ChinmayMhatre) [inline mehmetsaidozsoy.json:21]:
```suggestion
      "icon": "FaGithub"
```
You could use the GitHub icon for your GitHub profile

File: mehmetsaidozsoy.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      "name": "Follow me on LİnkedIn",
+      "url": "https://www.linkedin.com/in/mehmet-said-özsoy-7b844513/",
+      "icon": "FaInstagram"
```
Comment (ChinmayMhatre) [inline mehmetsaidozsoy.json:26]:
```suggestion
      "icon": "FaLinkedin"
```
There is a linked in icon aswell

File: mehmetsaidozsoy.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      "name": "Follow me on Youtube",
+      "url": "https://youtube.com/@MehmetSaidOZSOY",
+      "icon": "FaInstagram"
```
Comment (ChinmayMhatre) [inline mehmetsaidozsoy.json:31]:
```suggestion
      "icon": "FaYoutube"
```
Here is this YouTube icon

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Changes to a few icons

**Comment by ChinmayMhatre** [comment]
@mehmetsaidozsoy Please move you json file to the data folder

File: mehmetsaidozsoy.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    },
```
Comment (ChinmayMhatre) [inline mehmetsaidozsoy.json:32]:
```suggestion
    }
    ]
```
There is a missing closing bracket for the link

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: mehmetsaidozsoy.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Mehmet Said ÖZSOY",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": " Motorcycle lover",
+  "avatar": "https://avatars.githubusercontent.com/u/6255955?s=40&v=4",
```
Comment (ChinmayMhatre) [inline mehmetsaidozsoy.json:6]:
```suggestion
  "avatar": "https://github.com/mehmetsaidozsoy.png",
```
Please use this icon

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Your file must be inside the `data` directory.
Please move it there. Any help, let us know. 🙂

**Review by devvsakib — APPROVED** [review]

**Comment by SaraJaoude** [comment]
@mehmetsaidozsoy you seem to have created another Pull Request for your profile in https://github.com/EddieHubCommunity/LinkFree/pull/4649 which supersedes this one.  

I will be closing this Pull Request. In future it would be helpful if you could close your own Pull Request if they are no longer required, as this helps maintainers manage the project.

**Pull request closed without merging**